roles/kubernetes/node/templates/kubelet.j2

```diff
@@ -14,6 +14,8 @@ KUBELET_HOSTNAME="--hostname-override={{ ansible_hostname }}"
 --kube-reserved cpu={{ kubelet_cpu_limit }},memory={{ kubelet_memory_limit|regex_replace('Mi', 'M') }} \
 --node-status-update-frequency={{ kubelet_status_update_frequency }} \
 {% if kube_version | version_compare('v1.6', '>=') %}
+{# flag got removed with 1.7.0 #}
+{% if kube_version | version_compare('v1.7', '<') %}
 --enable-cri={{ kubelet_enable_cri }} --cgroups-per-qos={{ kubelet_cgroups_per_qos }} \
```

Comment:
anyhow, you should double check the file after the jinja2 replacements. i haven't checked it.